addons/mail/models/mail_user_settings.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class MailUserSettings(models.Model):
+    _name = 'mail.user.settings'
+    _description = 'Mail User Settings'
+
+    is_discuss_sidebar_category_channel_open = fields.Boolean(string="Is discuss sidebar category channel open?", default=True)
```

Comment:
Shoudln't we inverse the meaning ? Purpose would be to store as much False as possible, if you have lots of users, better have falsy than truthy by default if possible. (same for other field) As I am unsure about whether people will keep it open or not, I let you think about its usage.

Reply:
One of the goal of having a separate table is that we don't need to make a migration and we don't need to pre-fill it for all existing users.
It will be generated for each user on the fly, as he is connecting the first time, so I don't think it matters whether the value is True of False by default in this case.
I wouldn't want to have to do negation everywhere in the business code if it's avoidable.